pkg/proc/target.go

```diff
@@ -181,6 +181,19 @@ func NewTarget(p Process, currentThread Thread, cfg NewTargetConfig) (*Target, e
 	return t, nil
 }
 
+// IsCgo returns the value of runtime.iscgo
+func (t *Target) IsCgo() bool {
+	scope := globalScope(t.BinInfo(), t.BinInfo().Images[0], t.Memory())
```

Comment:
Seems we only call this once, but maybe we should memoize the result just in case?